### PR TITLE
Shueja personal/7/trajectory

### PR DESCRIPTION
**Pull request by shueja-personal**

### CODE REVIEW

**Comment by shueja-personal** [comment]
anylint is only failing on javadocs.

**Comment by AriShashivkopanazak** [comment]
Readme has anylint errors, I suggest getting a markdown linter and fixing everything

File: PathWeaver/Paths/Unnamed.path [hunk 1]
```diff
@@ -0,0 +1,3 @@
+X,Y,Tangent X,Tangent Y,Fixed Theta,Name
```
[the remark this answers is not in the document]
Comment (shueja-personal) [reply PathWeaver/Paths/Unnamed.path]: Deleted. we hard code the trajectories anyway

**Comment by shueja-personal** [comment]
Readme is outside the scope of this PR. I don't believe I touched it at all.
The separate constants file is exactly why I wanted to merge this change. We need to refactor out our constants file in Robot-2020 so it is more obvious which constants go where.

File: src/main/java/frc/designs/DrivebaseS.md [hunk 2]
```diff
@@ -25,15 +25,15 @@ Slave motors must be on Victor SPX
```
[the remark this answers is not in the document]
Comment (shueja-personal) [reply src/main/java/frc/designs/DrivebaseS.md]: This should be a change to the template also.

**Pull request merged**